README.md

```diff
@@ -92,7 +94,8 @@ This project doesn't need to be deployed
 
 ## 👥 Authors <a name="authors"></a>
 
-👤 **Collins Akorda**
+👤 **Collins Akorda, Maryam**
+    
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: add your partner details but you just add your partner name kindly add your partner details also :slightly_smiling_face: check below :point_down:  the example :+1: 

**Example**
![Screenshot from 2023-04-18 17-02-06](https://user-images.githubusercontent.com/45226848/232771185-0a8be3db-7665-4938-a781-60206ad69bc1.png)